src/OrchardCore.Modules/OrchardCore.Media.Indexing/Manifest.cs

```diff
@@ -0,0 +1,53 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Media Indexing",
+    Author = ManifestConstants.OrchardCoreTeam,
+    Website = ManifestConstants.OrchardCoreWebsite,
+    Version = ManifestConstants.OrchardCoreVersion
+)]
+
+[assembly: Feature(
+    Id = "OrchardCore.Media.Indexing",
+    Name = "Media Indexing",
+    Description = "Provides a way to index media files with common format in search providers.",
+    Dependencies =
+    [
+        "OrchardCore.Media"
+    ],
```

Comment:
```suggestion
    Dependencies = [ "OrchardCore.Media" ],
```

Reply:
I don't like the one liner which is why I span it on multiple lines on purpose.

Reply:
It was there if I'm wrong, we usually span into multiple lines if we have more than one or two, anyhow I think we're ready to merge this IMHO